docs/saml-2-0/README.md

```diff
@@ -9,6 +9,12 @@ In your IdP you can provide the following endpoint URLs to configure SSO, where
 - Single Logout Service (SLS): `https://<site-url>/sso/sls`
 - Assertion Consumer Service (ACS): `https://<site-url>/sso/verify`
 
+**Note:**: `<site-url>` will default to your primary network URL rather than the current site's URL. For single site mode use the following filter:
```

Comment:
Your comment on this PR mentions "per site mode", rather than "single site mode" (which implies non-multisite).

Reply:
Good call, should be less confusing.
```suggestion
**Note:**: `<site-url>` will default to your primary network URL rather than the current site's URL. For per site mode use the following filter:
```